README.md

```diff
@@ -12,14 +12,27 @@ We recommend using VSCode with the [Vala plugin](https://marketplace.visualstudi
 
 - Alpine Linux Edge: `apk add vala-language-server`
 
-- Ubuntu 18.04 and elementary:
+- Ubuntu 18.04 and elementaryOS
```

Comment:
Can you change this to:

> Ubuntu 18.04/19.10/20.04 and elementaryOS

I promise this is the last thing I'll ask for.

Reply:
I think for 19.10 and 20.04 this instruction is not relevant enough :)

Ubuntu 19.10 has `libvala-0.44-0` package and Ubuntu 20.04 has `libvala-0.48-0`, which is relevant for this project (I hope it is not going to change soon enough?)

Reply:
What if I make separate sections for 19.10 and 20.04? Or subsections specifically for 18.04 and 19.10?

Reply:
Rather than making specific sections, change the line:

> In order to install VLS, you need Vala 0.48 or later, which may be unavailable on your distribution.

to:

> In order to install VLS, you need Vala 0.48 or later, which is only available by default in Ubuntu 20.04.

Reply:
My point was that this line:
```
    sudo apt-get remove valac-0.40-vapi "libvala.*-0.40-0"
```

Is relevant only for 18.04 and eOS and not for 19.10. Though, Ubuntu 19.10 has end of life schedulet to July 2020... So I think it can be skipped for now.